### PR TITLE
test(api): add goerli to ease testing (#58)

### LINKED ISSUE #58: Add Goerli to ease testing
https://app.dework.xyz/nation3/develop-1?taskId=c90f5923-8092-41be-8689-dc0259747880

Needed in #56

### PR DESCRIPTION
**Pull request by aahna-ashina**
## Dework Task

https://app.dework.xyz/nation3/develop-1?taskId=c90f5923-8092-41be-8689-dc0259747880

## Related GitHub Issue

close #58 

## Admin Tasks

- [x] Add `NEXT_PUBLIC_CHAIN=mainnet` environment variable to Vercel deployment.

### CODE REVIEW

File: server/pages/api/downloadPass.ts [hunk 1]
```diff
@@ -51,9 +52,13 @@ export default function handler(req: NextApiRequest, res: NextApiResponse) {
     // TODO
 
     // Check that the address has a passport NFT
+    const passportIssuerAddress : string = (process.env.NEXT_PUBLIC_CHAIN == 'goerli') 
+        ? '0x8c16926819AB30B8b29A8E23F5C230d164337093' 
+        : '0x279c0b6bfCBBA977eaF4ad1B2FFe3C208aa068aC'
```
Comment (johnmark13) [inline server/pages/api/downloadPass.ts:57]: You've config'd so much, how about the addresses?

**Review by johnmark13 — APPROVED** [review]
Two small comments, but all looks good to me to move forward.

File: server/pages/api/downloadPass.ts (hunk 1 above)
Comment (aahna-ashina) [reply server/pages/api/downloadPass.ts]: @johnmark13 Fixed:  https://github.com/nation3/mobile-passport/pull/59/commits/e5e9a3f237dd0045d2d401992d05a157cc4dee01

Note that I wasn't able to use the Config in `_middleware.ts`, so I just left it as a `// TODO` there.

File: server/utils/Config.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  if (!variable) {
+    throw new Error(`Missing variable: "${environmentVariable}"`)
+  } else {
+    return variable
+  }
+}
+
+export const config = {
+  chain: getEnvironmentVariable('NEXT_PUBLIC_CHAIN'),
+  inpuraApiKey: getEnvironmentVariable('INFURA_ENDPOINT'),
```
Comment (johnmark13) [inline server/utils/Config.ts:13]: `infuraApikey`?

File: server/utils/Config.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  passportAddress: (getEnvironmentVariable('NEXT_PUBLIC_CHAIN') == 'goerli') ? '0x51F728c58697aFf9582cFDe3cBD00EC83E9ae7FC' : '0x3337dac9f251d4e403d6030e18e3cfb6a2cb1333',
+  passportIssuerAddress: (getEnvironmentVariable('NEXT_PUBLIC_CHAIN') == 'goerli') ? '0x8c16926819AB30B8b29A8E23F5C230d164337093' : '0x279c0b6bfCBBA977eaF4ad1B2FFe3C208aa068aC',
+  
+  appleCertificatePEM: getEnvironmentVariable('APPLE_CERTIFICATE_PEM'),
+  appleCertificateKey: getEnvironmentVariable('APPLE_CERTIFICATE_KEY'),
+
+  // TODO: add google variables
+  // googleApplicationCredentials: getEnvironmentVariable('GOOGLE_APPLICATION_CREDENTIALS'),
+  // googleWalletIssuerId: getEnvironmentVariable('GOOGLE_WALLET_ISSUER_ID'),
+  // googleWalletUserId: getEnvironmentVariable('GOOGLE_WALLET_USER_ID'),
```
Comment (johnmark13) [inline server/utils/Config.ts:24]: Maybe have these unusued variables (not commented out), or don't have them and just have the TODO. Not a big deal!

**Review by johnmark13 — APPROVED** [review]
One typo, one comment, otherwise I think this is all good!

**Review by luisivan — APPROVED** [review]

File: server/utils/Config.ts (hunk 2 above)
Comment (aahna-ashina) [reply server/utils/Config.ts:13]: > `infuraApikey`?

I don't understand?

**Pull request merged**